### PR TITLE
Fix pixilated images

### PR DESCRIPTION
**Pull request by nerdlet**
## Description

Fixes # (issue)

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

## Desktop Screenshots
Screenshot 2021-02-09 at 15 12 32

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/lib/wp/index.js [hunk 1]
```diff
@@ -133,42 +139,80 @@ function wp(site) {
...
+      featured_media: embedded["wp:featuredmedia"][0],
+      title: resource.title.rendered,
+      // set thumbail from acf | generated thumbnail | Featured Image
+      thumbnail_image:
+        resource.acf.attributes.thumbnail_image ||
+        resource.featured_image_src ||
+        embedded["wp:featuredmedia"][0] ||
+        null,
+    };
+    return post;
```
Comment (kilemensi) [inline src/lib/wp/index.js:168]:
```suggestion
    const featuredMedia: embedded["wp:featuredmedia"]?.length ? embedded["wp:featuredmedia"][0] : null;
    const post = {
      ...resource,
      author: embedded.author[0],
      content: resource.content.rendered,
      featured_media: featuredMedia,
      title: resource.title.rendered,
      // set thumbnail from acf | generated thumbnail | Featured Image
      thumbnail_image:
        resource.acf.attributes.thumbnail_image ||
        resource.featured_image_src ||
        featuredMedia,
    };
    return post;
```

What's the difference between `resource.featured_image_src` and the normal `wp:featuredmedia`?

**Review by kilemensi — APPROVED** [review]
🚀 @KobbyMmo 

---------------------------

I think the methods were arranged as in getPages, getPosts, etc., and looks like you're changing that.

**Comment by kilemensi** [comment]
> rocket @KobbyMmo
> 
> I think the methods were arranged as in getPages, getPosts, etc., and looks like you're changing that i.e. mixing getPost in getPages or something.

File: src/lib/wp/index.js (hunk 1 above)
Comment (KobbyMmo) [reply src/lib/wp/index.js:168]: @kilemensi 
featured_media_src, is a generated thumbnail from the featured media. If the user sets the thumbnail, we use it, if not we use a generated thumbnail from the featured image, if that's not available, we use the featured image.

>I think the methods were arranged as in getPages, getPosts, etc., and looks like you're changing that.

Lint was complaining that the getpostbySlug was used before it was called (I use it to get the posts from pages with posts)
So i sent the method above it.

Comment (kilemensi) [reply src/lib/wp/index.js:168]: Then move all `getPost` methods to before `getPage` onces... I think they were arranged alphabetically. I don't think any of the `getPost` method call `getPage`.

**Pull request merged**